src/app/team-tile/team-tile.component.html

```diff
@@ -1,13 +1,15 @@
 <div class="tile is-parent" (click)="incrementBtnClick($event)">
-  <article class="tile is-child notification is-{{ color }} {{ animatedClass }}">
-    <div class="content">
-      <!-- <p class="title">Tall tile</p> -->
-      <!-- <p class="subtitle">With even more content</p> -->
-      <div class="content">
-        <!-- Content -->
-        <h1 class="score has-text-centered has-text-white-bis">{{ points || 0}}</h1>
-      </div>
-    </div>
-  </article>
-</div>
-
+    <article class="tile is-child notification is-{{ color }} {{ animatedClass }}">
+        <div class="content">
+            <!-- <p class="title">Team {{ color }}</p> -->
+            <!-- <p class="subtitle">With even more content</p> -->
+            <div class="content">
+                <!-- Content -->
+                <h1 class="score has-text-centered has-text-white-bis is-unselectable">{{ points || 0}}</h1>
+            </div>
+        </div>
+        <div class="content" (click)="decrementBtnClick($event); $event.stopPropagation()">
```

Comment:
I think this should should be a button block instead.
- buttons: http://bulma.io/documentation/modifiers/syntax/
- block jazz: http://bulma.io/documentation/modifiers/responsive-helpers/